### PR TITLE
feat: [SRM-10806]: fixing the validation for condition field of Notifications.

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Fixing the validation for condition field of Notifications.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/strings/strings.en.yaml [hunk 1]
```diff
@@ -533,6 +533,8 @@ notifications:
   changeType: Change Type
   healthScoreBelow: Health score is below
   thresholdBelow: Threshold is below
+  validations:
+    conditionIsRequired: Condition is Required
```
Comment (upadhyay-an) [inline src/modules/85-cv/strings/strings.en.yaml:537]: nit: I think required should be in smallcase.

**Review by upadhyay-an — APPROVED** [review]

File: src/modules/85-cv/components/Notifications/NotificationsContainer.types.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export type NotificationConditions = StepProps<SRMNotification> & {
 export interface NotificationRule {
   id: string
   condition: SelectOption | null
-  changeType?: MultiSelectOption[]
+  changeType?: MultiSelectOption[] | { label: string; value: string }
```
Comment (amitsh14) [inline src/modules/85-cv/components/Notifications/NotificationsContainer.types.ts:42]: Nitpick: can { label: string; value: string } be declared separately and used here to improve the readability

**Review by amitsh14 — APPROVED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

File: src/modules/85-cv/components/Notifications/NotificationsContainer.types.ts (hunk 2 above)
Comment (harshilbits) [reply src/modules/85-cv/components/Notifications/NotificationsContainer.types.ts]: done.

File: src/modules/85-cv/strings/strings.en.yaml (hunk 1 above)
Comment (harshilbits) [reply src/modules/85-cv/strings/strings.en.yaml]: done.

**Review by amitsh14 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**